tests/page-objects/default/sms/messages.wdio.page.js

```diff
@@ -53,7 +53,7 @@ const sendMessageToPhone = async (message, phone) => {
 const searchSelect = async (searchText, entrySelector) => {
   await messageRecipientSelect().setValue(searchText);
   const elm = await $(entrySelector);
-  await elm.waitForDisplayed(4000);
+  await elm.waitForDisplayed(5000);
```

Comment:
I change this line of code locally to `await elm.waitForDisplayed();` run the test, and it passes. These time-based errors always happen, but when retrying, it works. I know is a problem but I don't think the solution is to increase the timeout. Can you try assigning more resources to your docker and rerun it?

Reply:
We took this to slack and the problem is not locally, is when running github-ci